doc/source/index.rst

```diff
@@ -36,6 +36,7 @@ Welcome to slurp's documentation!
    teresa.md
    al-latrun.md
    wasfi.md
+   steve-irwin.md
```

Comment:
The menu needs to be in alphabetical order by document title.

https://slurp--1093.org.readthedocs.build/en/1093/

<img width="289" alt="Screen Shot 2021-07-26 at 6 30 49 pm" src="https://user-images.githubusercontent.com/418747/126958645-3aa40ffa-2f81-4813-a648-ff372e00fab0.png">

Reply:
Pushed the changes.